dimagi/pages/templates/sections/commcare/partners/brief_overview.html

```diff
@@ -0,0 +1,52 @@
+{% load i18n %}
+{% load dimagidotcom %}
+
+{% url 'services' as services %}
+{% url 'commcare' as commcare %}
+{% url 'focus_mdm' as focus_mdm %}
+
+<section id="brief_overview"
+         data-lazybg="true">
+  <div class="content pad-top-medium pad-bottom-medium">
+    <div class="pad-bottom-line pad-bottom-small-at-medium pad-bottom-small-at-large pad-bottom-small-at-extra-large">
+      <h2 class="turquoise">
+        {% blocktrans %}
+          An Overview
+        {% endblocktrans %}
+      </h2>
+    </div>
+    <div class="grid grid-2-at-medium grid-1-at-small">
+      <div class="column">
+        <p>
+          {% blocktrans %}
+            From advising on digital strategy for frontline services to 
+            offering the world’s most powerful mobile data collection 
+            platform and a device management system to match, Dimagi’s 
+            partnerships cover a wide range of needs. With experience in 80+ 
+            countries working on more than 2,000 programs, our partners range 
+            from small businesses and nonprofits to ministries of health and 
+            multinational non-governmental organizations.
+          {% endblocktrans %}
+        </p>
+        <p>
+          {% blocktrans %}
+            Our world-class <a href="{{ services }}">Global Services</a> team 
+            has spent more than a decade developing mobile solutions for 
+            projects as small as 10-person pilots and as large as multi-country 
+            initiatives. And our software platforms, 
+            <a href="{{ commcare }}">CommCare</a> and 
+            <a href="{{ focus_mdm }}">FocusMDM</a>, help improve the 
+            efficiency, sustainability, and impact of their programs.
+          {% endblocktrans %}
+        </p>             
+      </div>{# .column #}
+      <div class="column" id="partners-intro">
+        <div class="intro-image">
+          <div class="hero-data-image white-lazy-bg"
+               data-lazybg="true">
+          </div>
```

Comment:
no need for a new line here (there are a number of places where this is the case. please only add a new line for a `div` if there's content inside